site/specs-temp/numbers.json

```diff
@@ -7296,6 +7296,222 @@
+            "in": "query",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            },
+            "example": 20
+          },
+          {
+            "name": "startdate",
+            "description": "Checks the order's creation date against this value. Orders that have a creation date after this date will be included. Format is yyyy-MM-dd",
+            "in": "query",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "example": "2013-10-22"
+          },
+          {
+            "name": "enddate",
+            "description": "Checks the order's creation date against this value. Orders that have a creation date before this date will be included. Format is yyyy-MM-dd",
+            "in": "query",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "example": "2013-10-25"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The response provides a list of all of the orders in the date range that have been issued against the account.\n",
+            "content": {
+              "application/xml": {
+                "examples": {
+                  "example": {
+                    "description": "example",
+                    "value": "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<ResponseSelectWrapper>\n   <ListOrderIdUserIdDate>\n      <TotalCount>3</TotalCount>\n      <OrderIdUserIdDate>\n         <accountId>14</accountId>\n         <CountOfTNs>2</CountOfTNs>\n         <userId>team_ua</userId>\n         <lastModifiedDate>2014-07-07T10:06:43.427Z</lastModifiedDate>\n         <OrderType>import_to_account</OrderType>\n         <OrderDate>2014-07-07T10:06:43.427Z</OrderDate>\n         <orderId>37a6447c-1a0b-4be9-ba89-3f5cb0aea142</orderId>\n         <OrderStatus>FAILED</OrderStatus>\n      </OrderIdUserIdDate>\n      <OrderIdUserIdDate>\n         <accountId>14</accountId>\n         <CountOfTNs>2</CountOfTNs>\n         <userId>team_ua</userId>\n         <lastModifiedDate>2014-07-07T10:05:56.595Z</lastModifiedDate>\n         <OrderType>import_to_account</OrderType>\n         <OrderDate>2014-07-07T10:05:56.595Z</OrderDate>\n         <orderId>743b0e64-3350-42e4-baa6-406dac7f4a85</orderId>\n         <OrderStatus>RECEIVED</OrderStatus>\n      </OrderIdUserIdDate>\n      <OrderIdUserIdDate>\n         <accountId>14</accountId>\n         <CountOfTNs>2</CountOfTNs>\n         <userId>team_ua</userId>\n         <lastModifiedDate>2014-07-07T09:32:17.234Z</lastModifiedDate>\n         <OrderType>import_to_account</OrderType>\n         <OrderDate>2014-07-07T09:32:17.234Z</OrderDate>\n         <orderId>f71eb4d2-bfef-4384-957f-45cd6321185e</orderId>\n         <OrderStatus>RECEIVED</OrderStatus>\n      </OrderIdUserIdDate>\n   </ListOrderIdUserIdDate>\n</ResponseSelectWrapper>"
```

Comment:
is there a reason you didn't capitalize `<OrderId>`?

Reply:
because for some reason
<img width="651" alt="Снимок экрана 2022-08-16 в 10 01 34" src="https://user-images.githubusercontent.com/30110721/184817603-53b5bb13-71d4-4a12-9b5f-2ce2f3210836.png">